### PR TITLE
Establish common JS and CSS

### PR DESCRIPTION
**Pull request by ChristopherTrimboli**
- Start a common JS for all Exokit sites.
- Separate common CSS from app specific CSS.

Benefit is to increase reusability and simplicity of further projects, using store as template.
Drag n drop functionality from site to site without tearing apart app specific code.
A lot of things are done the same way through all sites, verify, login, navbar.
Perfecting these and making them modular is a good practice that will speed up app development and make things easier and less error prone.

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Verify user's code, returns boolean.
```
Comment (avaer) [inline common.js:1]: This doesn't seem done; should we leave it for another PR?

File: common.css [hunk 2]
```diff
@@ -1,3 +1,16 @@
+/*  
+
+Exokit Common CSS Styles
+
+Color Pallete:
+
+#5c6bc0 - Exokit Blue
```
Comment (avaer) [inline common.css:7]: These could be CSS variables.

**Review by avaer — APPROVED** [review]

File: common.js (hunk 1 above)
Comment (ChristopherTrimboli) [reply common.js:1]: No, just was using this as example code for refactor. I'll fill these in in new PR.
